packages/sui-domain/src/EntryPointFactory.js

```diff
@@ -1,14 +1,15 @@
 import createNotImplementedUseCase from './createNotImplementedUseCase.js'
 
-export default ({useCases, config}) =>
+export default ({useCases, config, logger}) =>
   class EntryPoint {
     subscribers = {}
 
-    constructor(params = {config: {}}) {
+    constructor(params = {config: {}, logger: {}}) {
```

Comment:
Maybe would be nice to add that you can now inject the logger in the `README` file and some example of how it can be used 🤔

Reply:
Agree

Reply:
Done! ✅
Let me know if we can improve it 🙏🏻  @rmoralp @andresz1